spec/system/candidate_interface/candidate_entering_personal_details_spec.rb

```diff
@@ -77,4 +87,12 @@ def then_i_can_check_my_answers
   def then_i_can_see_my_application
     expect(page).to have_content t('page_titles.application_form')
   end
+
+  def then_i_should_see_validation_errors
+    expect(page).to have_content t('activemodel.errors.models.candidate_interface/personal_details_form.attributes.first_name.blank')
+    expect(page).to have_content t('activemodel.errors.models.candidate_interface/personal_details_form.attributes.last_name.blank')
+    expect(page).to have_content t('activemodel.errors.models.candidate_interface/personal_details_form.attributes.date_of_birth.invalid')
+    expect(page).to have_content t('activemodel.errors.models.candidate_interface/personal_details_form.attributes.first_nationality.blank')
+    expect(page).to have_content t('activemodel.errors.models.candidate_interface/personal_details_form.attributes.english_main_language.blank')
```

Comment:
Could this be made more readable using `scope:`? Eg

```
scope = 'activemodel.errors.models.candidate_interface/personal_details_form.attributes'
expect(page).to have_content t('first_name.blank', scope: scope)
expect(page).to have_content t('last_name.blank', scope: scope)
```

Reply:
Oh yes <3